rust/minimal-counter-dapp/src/minimal_rust_dapp/lib.rs

```diff
@@ -0,0 +1,26 @@
+
+
+
+static mut COUNTER: u64 = 0;
+
+#[ic_cdk_macros::update]
+fn increment() -> u64 {
+    unsafe {
+        COUNTER += 1;
+        COUNTER
+    }
+}
```

Comment:
We should be able to do this without `unsafe`.

A couple options:
- a RefCell (example: https://github.com/dfinity/cdk-rs/blob/main/examples/counter/src/counter_rs/lib.rs)
- a static AtomicU64